src/frontend/src/pages/AdminToolsPage/TeamsTools.tsx

```diff
@@ -79,7 +81,11 @@ const TeamsTools = () => {
 
   const teamTableRows = allTeams.map((team) => (
     <TableRow>
-      <TableCell sx={{ border: '2px solid black' }}>{team.teamName}</TableCell>
+      <TableCell sx={{ border: '2px solid black' }}>
```

Comment:
clicking anywhere on the row should link to the team page, not just this cell

Reply:
Just pushed another commit with the fix. Sorry about that.